new_mvc_local/controllers/equipController.js

```diff
@@ -343,3 +343,55 @@ exports.getFixtureHistoryByID = async (req, res) => {
 
 
 }
+
+
+exports.fixturesMovement = async (req, res) => {
+
+    let fixtureRow = [];
+
+    console.log("fixturesMovement");
+    console.log("req.body:", req.body);
+
+    let status = await auth.authenticateJWT(req, res);
+    console.log("statusCode:", status);
+
+
+    if (status.status === 200) {
+
+        // fixtureRow.push(req.body.idFixture);
+        // fixtureRow.push(req.body.idAction);
+        // fixtureRow.push(req.body.comments);
+        // fixtureRow.push(req.body.spareParts);
+        // fixtureRow.push(req.body.date);
+        // fixtureRow.push(req.params.status);
+        // fixtureRow.push(Date.now());
+        // fixtureRow.push(req.body.idEvent);
+
+        // console.log("fixtureRow:", fixtureRow);
+
+        let idFixture = [];
+
+        for (let i = 1; i < req.body.length; i++) {
+            idFixture.push(req.body[i].idFixture);
```

Comment:
see if you can use map, `const idFixture = req.body.map(item => item.idFixture)`